app/server.py

```diff
@@ -18,6 +18,46 @@ def index():
 ######################################################################
 @app.route('/shopcarts/<int:id>', methods=['GET'])
 def get_shopcarts(id):
+    """
+    Get a Shopcart
+    This endpoint will get a shopcart.
+    ---
+    tags:
+      - Shopcarts
+    produces:
+      - application/json
+    parameters:
+      - name: user_id
+        in: path
```

Comment:
If I try this out as is I always get a 404. Swagger makes you match a path parameter with the actual param used in Flask (see line 19), so here it you have to use the param name `id`, and similarly for line 41.

Reply:
lol did notttt see @jonjrodriguez's review till now